testing/code/test_excinfo.py

```diff
@@ -1352,6 +1352,18 @@ def unreraise():
         )
         assert out == expected_out
 
+    def test_exec_type_error_filter(self, importasmod):
```

Comment:
```suggestion
    def test_exec_type_error_filter(self, importasmod):
        """traceback filtering in presence of non-dicts as globals/locals (#7742)"""
```

Mostly to add the issue number, as I find that useful on occasion when trying to figure out what a test is really testing.

Reply:
sure, though `git log -G` or `git blame` could get the same info

Reply:
> sure, though git log -G or git blame could get the same info

True, but often code moves around and makes it harder to track down (however it is possible, I agree).

Reply:
I shortened the docstring btw **shakesfist pydocstyle**